apps/studio/src/features/editing-experience/components/FormBuilder.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  FormControl,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInputField,
+  NumberInputStepper,
+  RadioGroup,
+  Select,
+} from '@chakra-ui/react'
+import {
+  Button,
+  FormLabel,
+  Input,
+  NumberInput,
+  Radio,
+  Switch,
+} from '@opengovsg/design-system-react'
+import { IsomerComponent } from '@opengovsg/isomer-components'
+import { useForm } from 'react-hook-form'
+import IsomerSchema from './0.1.0.json'
```

Comment:
eventually we need to fetch this at runtime from schema.isomer.gov.sg but for dev, can use this first

Reply:
pls add a `TODO`